src/index.js

```diff
@@ -0,0 +1,127 @@
+import './css/reset.css';
+import './css/style.css';
+import './css/list_projects.css';
+import './css/list_todos.css';
+import './css/forms.css';
+
+import content from './js/content';
+import listProjects from './js/list_projects';
+import listTodos from './js/list_todos';
+import formTodo1 from './js/form_todo';
+import formProject1 from './js/form_project';
+
+import Project from './js/project';
+import Todo from './js/todo';
+
+
+const App = JSON.parse(localStorage.getItem('appData')) || [];
+
+let currentProject = 0;
+
+window.changeTodosList = function (index) {
+  listTodos(App, index);
+  currentProject = index;
+
+  listProjects(App, currentProject);
+};
+
+window.deleteProject = function (index) {
+  App.splice(index, 1);
+
+  localStorage.setItem('appData', JSON.stringify(App));
+
+  listProjects(App, currentProject);
+
+  listTodos(App, 0);
+};
+
+window.deleteTodo = function (index) {
+  App[currentProject].todos.splice(index, 1);
+
+  localStorage.setItem('appData', JSON.stringify(App));
+
+  listTodos(App, currentProject);
+};
+
+window.toggleStatus = function (index) {
+  App[currentProject].todos[index].toggleStatus();
+
+  localStorage.setItem('appData', JSON.stringify(App));
+};
+
+window.formProject = function () {
+  formProject1();
+};
+
+window.formTodo = function (index = -1) {
+  formTodo1(App, currentProject, index);
+};
+
+function hideForm() {
+  const container = document.getElementById('container');
+  container.classList.add('none');
+
+  const formContainer = document.getElementById('form_container');
+  formContainer.classList.add('none');
+}
+
+function includes(value) {
+  for (let i = 0; i < App.length; i += 1) {
+    if (App[i].title === value) return true;
+  }
+
+  return false;
+}
+
+window.saveProject = function (title, desc) {
+  if (includes(title)) {
+    alert('There is a project with the same name');
```

Comment:
- [ ] As the last reviewer mentioned you should not use alert() methods in today's web. That's considered a bad practice and should be avoided. Please show the message in the DOM instead. You can create an element that only shows up when there's an error or warning like the rails flash messages.
[Check this StackOverflow thread about it.](https://stackoverflow.com/questions/20359807/why-shouldnt-i-use-alert-in-javascript)